corgi/tasks/sca.py

```diff
@@ -86,58 +108,37 @@ def save_component(
     if component["type"] not in Component.Type.values:
         raise ValueError("Tried to save component with unknown type: %s", component["type"])
 
-    component_type = component["type"].lower()
-    syft_purl = meta.get("purl")
-    if not syft_purl:
-        # TODO: Needs a lot more work and tests
-        # But this fixes the most basic issues for now
-        qualifiers = {}
-
-        version = meta.get("version", "")
-        release = meta.get("release")
-        if version and release:
-            version = f"{version}-{release}"
-
-        arch = meta.get("arch")
-        if arch:
-            qualifiers["arch"] = arch
-        # TODO: Might need a epoch qualifier
-
-        syft_purl = PackageURL(
-            type=component_type,
-            name=meta.get("name", ""),
-            version=version,
-            qualifiers=qualifiers,
-        ).to_string()
-
     created = False
     name = meta.pop("name", "")
+    # TODO: Do we need to add epoch to the unique constraint?
+    epoch = meta.pop("epoch", 0)
     version = meta.pop("version", "")
     release = meta.pop("release", "")
     arch = meta.pop("arch", "noarch")
 
     namespace = Brew.check_red_hat_namespace(component["type"], version)
-    if namespace == Component.Namespace.REDHAT:
```

Comment:
This is now handled by Keith's changes in https://github.com/RedHatProductSecurity/component-registry/pull/626 ?